doc/start/hardware-recommendations.rst

```diff
@@ -59,8 +59,41 @@ configured to consume.  We recommend 1 GB as a minimum for most systems.  See
 OSDs (ceph-osd)
 ---------------
 
-By default, OSDs that use the BlueStore backend require 3-5 GB of RAM.  You can
-adjust the amount of memory the OSD consumes with the ``osd_memory_target`` configuration option when BlueStore is in use.  When using the legacy FileStore backend, the operating system page cache is used for caching data, so no tuning is normally needed, and the OSD memory consumption is generally related to the number of PGs per daemon in the system.
+Memory
+======
+
+Bluestore uses it's own memory to cache data rather than relying on the
+operating system page cache.  In bluestore you can adjust the amount of memory
+the OSD attempts to consume with the ``osd_memory_target`` configuration
+option.  The OSD can only control how much memory is mapped rather than how
+much RSS memory it actually uses.  The kernel is not guaranteed to reclaim
+unmapped memory within any given time frame.  When Transparent Huge Pages are
```

Comment:
I would move this caveat discussion about THP to after the basics ("set the osd_memory_target option").  And also didn't we add an option (on by default) that disables this automatically?